public/pages/Snapshots/components/ErrorModal/ErrorModal.tsx

```diff
@@ -5,24 +5,27 @@
 
 import { EuiModal, EuiText, EuiButton, EuiModalHeader, EuiModalFooter, EuiModalBody, EuiModalHeaderTitle } from "@elastic/eui";
 import React from "react";
+import { RestoreError } from "../../../../models/interfaces"
 
 
 interface ErrorModalProps {
-  error: React.ErrorInfo;
-  onClick: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  error: RestoreError;
+  snapshotId: string;
+  onClick: (event?: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.KeyboardEvent<HTMLDivElement> | undefined) => void;
```

Comment:
"| undefined" is not required since we have specified `?`

Reply:
I removed the ? from each of those, leaving the | undefined. in next (small) PR